README.md

```diff
@@ -11,33 +11,30 @@ Our roadmap for the ReactJS Boilerplate is where you can learn about what featur
     - Optimisation of Minify code in production mode
 - Coding Standards and Guidelines (Statically analyzes our codes)
   - [ ] ESLint with our rules
-  - [ ] Prettier
-- Testing framework
-  - [x] Jest
-  - [ ] React Testing Library
-  - [ ] React Hooks Testing Library
-  - [ ] Storybook
-    - [x] base
-    - [ ] msw
-    - [ ] state management
-    - [ ] router
-    - [ ] css
-    - [ ] i18n
-  - [ ] E2E
-    - [x] Playwright
-    - [ ] Cypress
+  - [x] Prettier
 - Mock API server
   - [x] msw
 
+### Testing
+
+- Visual regression testing
+  - [x] Storybook
```

Comment:
@jinmayamashita I think we should untick Storybook to implement later

Reply:
@mvn-luanvu-hn I agree. https://github.com/monstar-lab-oss/reactjs-boilerplate/pull/180/commits/92a8f7cb3a9a633d8528220e09876fa046bfc0ef in done.